packages/swap/src/utils/transactionRef.ts

```diff
@@ -0,0 +1,26 @@
+import { CHARSET as BASE58_CHARSET } from '@chainflip/utils/base58';
+import { Chain } from '@/shared/enums';
+
+const hexRegex = /^(0x)?[a-f0-9]+$/i;
```

Comment:
can simplify the checks below:
```suggestion
const hexRegexNoPrefix = /^[a-f0-9]+$/i;
const hexRegexWithPrefix = /^0x[a-f0-9]+$/i;
```